recipe/recipe.yaml

```
@@ -0,0 +1,84 @@
+schema_version: 1
+
+context:
+  python_min: 3.10
+  name: sp-repo-review
+  version: 2025.01.22
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/scientific-python/cookie/archive/refs/tags/${{ version }}.tar.gz
+  sha256: e4d0d54d07b608d825916e66185c7aefe05eceb090cb777e722f6e019f74ff4e
+
+build:
+  number: 1
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vvv
+
+requirements:
+  host:
+    - hatchling
+    - hatch-fancy-pypi-readme
+    - hatch-vcs
+    - pip
+    - python 3.10.*
```

```suggestion
    - python ${{ python_min }}.*
```

Bug: https://github.com/conda-incubator/conda-recipe-manager/issues/307